process-automation/latest/model/forms.md

```diff
@@ -186,6 +186,32 @@ The advanced properties for a REST dropdown field are:
 | Path to array in JSON response | The path to the JSON response. Enter `.` to use the full path. |
 | ID property | The ID of the REST service. |
 | Label property | The name of the REST service. |
+| Conditional | Turn this option on if you would like to link your drop down list with another drop down list and to create a conditional relationship between them. For example, if you select a country from one drop down list, the second drop down list will only show cities from that country. |
+| Depends on | Select which other dropdown list you would like to connect with. |
+| If equal | Select which child entry of the **Depends on** field you want to work with and add subordinate entries for it. |
+
+To create a conditional relationship between two dropdown fields using Country and City as an example:
+
+1. Add two dropdown list elements to your form.
+
+2. Name one of them Country and the other one City.
+
+3. Select the Country dropdown list element and click the **Advanced** tab.
+
+4. Select **Manual** and turn on the **Conditional** field.
+
+5. Select City from the **Depends on** dropdown list.
+
+6. Select the City dropdown list element and click the **Advanced** tab.
+
+7. Add a name for the label and then add all the Countries using the **Add option** field.
+
+8. Select the Country dropdown list element again and click the **Advanced** tab.
+
+9. From the **If equal** drop down list select the Country you want to work with and then add all the cities you would like available in the drop down list.
```

Comment:
same here all those steps configuring the lists are done in the child dropdown `City`

Reply:
Done